app/templates/app/all-installments.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'base.html' %}
+<!--{% load bootstrap4 %}-->
+<!--{% bootstrap_css %}-->
+{% block title %} Installments {% endblock title %}
+{% block content %}
+<html>
+  <head>
+    <link rel="stylesheet" href="../../static/css/installment_table.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.0/css/all.css">
+    <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.1/themes/base/jquery-ui.css" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
```

Comment:
If you are doing this in every template you should consider putting this in a separated template and `include` it. https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#include

Reply:
Yes, you are right, we will do it i other pull request.